server/index.js

```diff
@@ -0,0 +1,26 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+
+const path = require('path');
+const Stock = require('../database/Stock.js');
+
+const app = express();
+const port = 3000;
+
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({ extended: true }));
+
+app.listen(port, () => console.log('APP IS LISTENING'));
```

Comment:
@wwong007 @iamjaytong Consider logging what port it's listening on. Comes in handy.